### DISCUSSION TITLE
Can´t connect to Prometheus container in the same docker compose network

### DISCUSSION THREAD

**Discussion by lew-cmd** [perses/perses#1921] [Q&A] [answered]
Hi Team,

I am currently trying Perses as an alternative to Grafana for visualizing our metric data. First tries, migration of Grafana dashboards and such have been fruitful. Running Perses in a Docker container from my localhost, and connecting it to Prometheus Container instance on a different server does work - the datasource is configured as in the picture below. 
![image](https://github.com/perses/perses/assets/65823310/ec9c5db4-aab3-4986-929d-147fc054a415)

But when I am running the Perses container in the same network as the Prometheus instance (container name here is 'monitoring'), Perses is not able to fetch the data. 

![image](https://github.com/perses/perses/assets/65823310/dd226469-77e5-481a-a6e2-418cba59d197)

I tried to check connectivity from inside the container, but percli doesn´t seem to support 'curl' commands or similiar. Is that a known issue, and is there are workarround? 

Support here is appreciated a lot, and keep up the great work!

**Comment by celian-garcia** [comment, marked as answer]
Hello, thanks for trying Perses! 
I think this is a dupe of that https://github.com/perses/perses/issues/1417

Long story short direct access means that your browser will try to connect. Either use localhost:9090 if you expose that port in your docker-compose, or use Proxy mode
